src/main/java/com/example/BillingSystem/service/ProductService.java

```diff
@@ -16,15 +16,19 @@ public interface ProductService {
 
     Product saveProduct(Product product);
 
+    Product getProductDetails(String productType);
+
     Double calculateTotalPrice();
 
     Double calculateTotalTax();
 
+    Double calculateGrossPrice();
+
     Double[] getBill();
```

Comment:
Are we really using this getBill() method anywhere? If not, we can delete it from both interface and implemntation